docs/content/preview/api/ysql/the-sql-language/statements/ddl_alter_table.md

```diff
@@ -67,6 +67,37 @@ Renaming a table is a non blocking metadata change operation.
 {{< /note >}}
 
 
+#### SET TABLESPACE *tablespace_name*
+
+Asynchronously change the tablespace of an existing table. 
+The tablespace change will immediately reflect in the config of the table, however the tablet move by the load balancer happens in the background. 
+While the load balancer is performing the move it is perfectly safe from a correctness perspective to do reads and writes, however some query optimization that happens based on the data location may be off while data is being moved.
+
+{{< note title="Note" >}}
+
+This is a BETA feature, and we do not recommend running this command on a production database. You will see a warning when running the command but you can ignore it. Here is an example below:
```

Comment:
This is no longer a beta feature, so we can remove this warning

Reply:
Fixed in https://github.com/yugabyte/yugabyte-db/pull/20562/commits/b6623939e056e04feba0e89ee7f5fa8a2ebd3a09